### PR TITLE
Some missing impl

### PR DESCRIPTION
**Pull request by cdalexndr**
Depends on https://github.com/nebula-contrib/nebula-jdbc/pull/45

### CODE REVIEW

File: src/main/java/com/vesoft/nebula/jdbc/NebulaConnection.java [hunk 1]
```diff
@@ -282,7 +282,7 @@ public void rollback() throws SQLException {
 
     @Override
     public DatabaseMetaData getMetaData() throws SQLException {
-        throw ExceptionBuilder.buildUnsupportedOperationException();
+        return new NebulaDatabaseMetaData(this);
```
Comment (cdalexndr) [inline src/main/java/com/vesoft/nebula/jdbc/NebulaConnection.java:285]: Used by spring jdbc https://github.com/spring-projects/spring-framework/blob/3c42363ba49fdaf7f77f2bd135ecdf55167b982b/spring-jdbc/src/main/java/org/springframework/jdbc/core/StatementCreatorUtils.java#L263-L265

File: src/main/java/com/vesoft/nebula/jdbc/statement/NebulaPreparedStatementImpl.java [hunk 2]
```diff
@@ -281,7 +336,7 @@ public void setUnicodeStream(int parameterIndex, InputStream x, int length) thro
 
     @Override
     public void setNull(int parameterIndex, int sqlType) throws SQLException {
-        throw  ExceptionBuilder.buildUnsupportedOperationException();
+        insertParameter(parameterIndex, null);
```
Comment (cdalexndr) [inline src/main/java/com/vesoft/nebula/jdbc/statement/NebulaPreparedStatementImpl.java:342]: Used by spring jdbc: https://github.com/spring-projects/spring-framework/blob/3c42363ba49fdaf7f77f2bd135ecdf55167b982b/spring-jdbc/src/main/java/org/springframework/jdbc/core/StatementCreatorUtils.java#L282-L289

**Comment by Nicole00** [comment]
@cdalexndr Hi, there are conflicting files, could you please resolve it?

**Comment by cdalexndr** [comment]
@Nicole00 done

**Pull request merged**